app/views/petitions/check_results.html.erb

```diff
@@ -1,23 +1,22 @@
 <% content_for(:page_title) { "Create a new e-petition - e-petitions" } %>
-<% content_for :title do %>
-<div class="title_block">
-  <h1>Create a new e-petition</h1>
-</div>
-<% end %>
 
-<%= render "pre_creation_search_form" %>
+<%= link_to check_petitions_path, class: "back-page" do %>
+  &#9664; Back <span class="visuallyhidden"> to previous page</span>
```

Comment:
We took the approach from the register to vote app.
```
<a class="back-to-previous" href="#">
  Back <span class="visuallyhidden"> to the previous question</span>
</a>
```
Good point about the difference between `a` and `input` though. We either remove it or make "Back to previous page" more useful like "Back to petition details"